guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -219,6 +219,161 @@ Create the host from an image with the `hammer host create` command and include
+
+.VMware cloud-init Provisioning Overview
+
+When you set up the compute resource and images for VMware provisioning in {Project}, the following sequence of provisioning events occur:
+
+* The user provisions one or more virtual machines using the {Project} web UI, API, or hammer
+* {Project} calls the VMware vCenter to clone the virtual machine template
+* {Project} `userdata` provisioning template adds customized identity information
+* When provisioning completes, the {Project} `cloud-init` provisioning template instructs the virtual machine to call back to {SmartProxy} when `cloud-init` runs
+* VMware vCenter clones the template to the virtual machine
+* VMware vCenter applies customization for the virtual machine's identity, including the host name, IP, and DNS
+* The virtual machine builds, `cloud-init` is invoked and pings {Project} on port `80`, which then redirects to `443`
+
+.Port and Firewall Requirements
+
+Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}. Ensure that you configure port and firewall settings to open any necessary connections.
+
+ifeval::["{build}" == "satellite"]
+For more information about port and firewall requirements, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.4/html/installing_satellite_server_from_a_connected_network/preparing_your_environment_for_installation#ports_prerequisites[Port and Firewall Requirements] in the _Installation Guide_.
+endif::[]
+
+.Associating the `userdata` and `Cloud-init` Templates with the Operating System
+
+. In the {Project} web UI, navigate to *Hosts* > *Operating Systems*, and select the operating system that you want to use for provisioning.
+. Click the *Template* tab.
+. From the *Cloud-init template* list, select *Cloudinit default*.
+. From the *User data template* list, select *UserData open-vm-tools*.
+. Click *Submit* to save the changes.
+
+.Preparing an Image to use the cloud-init Template
+
+To prepare an image, you must first configure the settings that you require on a virtual machine that you can then save as an image to use in {Project}.
+
+To use the `cloud-init` template for provisioning, you must configure a virtual machine so that `cloud-init` is installed, enabled, and configured to call back to {ProjectServer}.
+
+ifeval::["{build}" == "satellite"]
+For security purposes, you must add a Satellite X509 certificate to use HTTPs for all communication. This procedure includes steps to clean the virtual machine so that no unwanted information transfers to the image you use for provisioning.
+endif::[]
+
+If you have an image with `cloud-init`, you must still follow this procedure to enable `cloud-init` to communicate with {Project} because `cloud-init` is disabled by default.
+
+. On the virtual machine that you use to create the image, install `cloud-init`, `open-vm-tools`, and `perl`:
```

Comment:
These instructions are for Red Hat Enterprise Linux or Fedora OS, follow similar steps for other Linux distributions.